backends/libParanumal/ceed-libparanumal.c

```diff
@@ -28,12 +28,12 @@ static int CeedInit_libparanumal(const char *resource, Ceed ceed) {
 
   ierr = CeedSetBackendFunction(ceed, "Ceed", ceed, "OperatorCreate",
                                 CeedOperatorCreate_libparanumal); CeedChk(ierr);
-  ierr = CeedSetBackendFunction(ceed, "Ceed", ceed, "OperatorDestroy",
-                                CeedOperatorDestroy_libparanumal); CeedChk(ierr);
+  // ierr = CeedSetBackendFunction(ceed, "Ceed", ceed, "OperatorDestroy",
```

Comment:
What trouble are you having with the interface - I would like to know for my PR that created the setter.

Reply:
OperatorDestroy and QFunctionDestroy are associated with the Operator and the QFunction, not the Ceed